cmd/runtimecfg/node-ip.go

```diff
@@ -135,6 +136,27 @@ func set(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+
+	// node ip hint for all other services
+	nodeIpOverrideDir := filepath.Dir(nodeIpFile)
+	err = os.MkdirAll(nodeIpOverrideDir, 0755)
+	if err != nil {
+		return err
+	}
+	log.Infof("Opening node ip override path %s", nodeIpFile)
+	nodeIpOverride, err := os.Create(nodeIpFile)
+	if err != nil {
+		return err
+	}
+	defer nodeIpOverride.Close()
+
+	nodeIpOverrideContent := fmt.Sprintf("NODE_IP=%s\n", nodeIP)
```

Comment:
I would prefer to just write the IP, rather than make this a systemd service-specific format. configure-ovs can still easily extract the IP from the file and it makes it easier for anything else that needs to consume this.

The goal here was to stop writing service-specific override files and just provide a single source of truth for the node ip.